scanner.go

```diff
@@ -21,24 +22,30 @@ func findFiles(rootPath string, excludes []string) <-chan ArchiveResult {
 		defer close(c)
 		err := filepath.Walk(rootPath, func(path string, fi os.FileInfo, err error) error {
 			if err != nil {
-				if os.IsNotExist(err) {
-					return nil
+				c <- ArchiveResult{&Archive{Path: path}, err}
+				if fi != nil && fi.IsDir() {
+					return filepath.SkipDir
 				}
+
+				// if errors.Is(err, fs.ErrPermission) ||
```

Comment:
what about those comments?

Reply:
Well.. I left them there on purpose to make it clear that we would also `return nil` for these error-cases. Can ofc remove them if desired